pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -306,10 +328,67 @@ func TestReconcile(t *testing.T) {
 		})
 	})
 
+	t.Run("create nstmplset OK", func(t *testing.T) {
+		t.Run("create", func(t *testing.T) {
+			r, req, _ := prepareReconcile(t, username, userAcc, preexistingUser, preexistingIdentity)
+
+			// test
+			_, err := r.Reconcile(req)
+
+			require.NoError(t, err)
+			checkStatus(t, r.client, username, "Provisioning", "")
```

Comment:
Instead of this, you could use already implemented custom assertion:
```
useraccount.AssertThatUserAccount(t, username, r.client).
				hasConditions(toBeNotReady("Provisioning", "")).
```
see eg. here https://github.com/codeready-toolchain/host-operator/blob/master/pkg/controller/masteruserrecord/masteruserrecord_controller_test.go#L70-L78